src/overlays/actors/ovl_En_Warp_tag/z_en_warp_tag.c

```diff
@@ -208,10 +208,9 @@ void EnWarpTag_RespawnPlayer(EnWarptag* this, PlayState* play) {
 
                 // why are we getting player home rotation from the room data? doesnt player have home.rot.y?
                 // especially because we are converting from deg to binang, but isnt home.rot.y already in binang??
-                Play_SetRespawnData(
-                    &play->state, 0, entrance, // parameter 3 is called "sceneSetup"
-                    play->setupEntranceList[playerSpawnIndex].room, playerParams, &newRespawnPos,
-                    ((((playerActorEntry->rot.y >> 7) & 0x1FF) / 180.0f) * 32768.0f)); // DEG_TO_BINANG ?
+                Play_SetRespawnData(&play->state, 0, entrance, // parameter 3 is called "sceneSetup"
```

Comment:
Above this function:
```c
// why are we getting player home rotation from the room data? doesnt player have home.rot.y?
// especially because we are converting from deg to binang, but isnt home.rot.y already in binang??
```
Given the recent player docs, do we happen to have an answer to this?

Reply:
If memory serves this is to do with the default camera stuff.